src/renderer/components/dock/terminal.ts

```diff
@@ -199,6 +200,15 @@ export class Terminal {
       }
     }
 
+    //Ctrl+K: clear the entire buffer, making the prompt line the new first line on mac os
+    if (isMac && metaKey) {
```

Comment:
Noticed that `Ctrl+L` clears output on Mac. Do we need one more hotkey?

Reply:
Yes, it did. But according to the comment, it would be nice to have the ability to "clear the scrollback buffer as well as the screen"
https://github.com/lensapp/lens/issues/2214#issuecomment-787975216

Reply:
@ixrock See https://github.com/lensapp/lens/issues/2214#issuecomment-787969824